README.rst

```diff
@@ -0,0 +1,33 @@
+ansible-openwisp2
+=================
+
+.. image:: https://github.com/openwisp/ansible-openwisp2/workflows/Ansible%20OpenWISP2%20CI%20Build/badge.svg?branch=master
+    :target: https://github.com/openwisp/ansible-openwisp2/actions?query=workflow%3A%22Ansible+OpenWISP2+CI+Build%22
+    :alt: Build Status
+
+.. image:: http://img.shields.io/badge/galaxy-openwisp.openwisp2-blue.svg?style=flat-square
+    :target: https://galaxy.ansible.com/ui/standalone/roles/openwisp/openwisp2/
+    :alt: Galaxy
+
+.. image:: https://img.shields.io/gitter/room/nwjs/nw.js.svg
+    :target: https://gitter.im/openwisp/general
+    :alt: Chat
+
+Ansible role that installs the OpenWISP Server Application.
+
+Tested on **Debian (Bookworm/Bullseye)**, **Ubuntu (24/22/20 LTS)**.
+
+**Recommended minimum ansible core version**: 2.13.
+
+Documentation
+-------------
+
+- `Developer documentation
+  <https://openwisp.io/docs/__new__/dev/ansible-openwisp2/developer/>`_
+- `User documentation
+  <https://openwisp.io/docs/__new__/dev/ansible-openwisp2/>`_
```

Comment:
Change the links to pretend it's already published, so I can just merge when ready to publish, do this on all modules please.